doc/user/content/sql/create-sink.md

```diff
@@ -95,6 +105,19 @@ the environment variable's presence to boot Materialize.
 `sasl_kerberos_principal` | `text` | Materialize Kerberos principal name. Required if `sasl_mechanisms` is `GSSAPI`.
 `sasl_kerberos_service_name` | `text` | Kafka's service name on its host, i.e. the service principal name not including `/hostname@REALM`. Required if `sasl_mechanisms` is `GSSAPI`.
 
+### `FORMAT`
+
+The data format of the emitted records.
+
+{{< diagram "sink-format-spec.svg" >}}
+
+### Consistency `FORMAT`
+
+The data format of the consistency records, if applicable.
+
+{{< diagram "consistency-format-spec.svg" >}}
```

Comment:
So, something is a little funny here (and is already funny with the `CREATE SINK` docs). In other places we don't intersperse the syntax with the description of that syntax. @sploiselle any thoughts? Doesn't need to block this PR.

Reply:
Yeah, this is odd and unwieldy. I would say, at the very least, we need to express which options are valid in these different contexts, though. Which is maybe? different than we do elsewhere, but I think having the variable names (e.g. those shown in the railroad diagrams) as close as possible to the the description of the variable is good. Think what looks odd here is the lack of description of the variables.

Reply:
I will move this around! Sorry it is awkward! I ended up breaking this diagram into smaller chunks because it was 2 to 3x the width of the docs page as a single item.